test/models/ui/UIComponentsSpec.scala

```diff
@@ -17,105 +17,55 @@
 package models.ui
 
 import base.{BaseSpec, TestConstants}
-import play.api.i18n.Lang
 
 class UIComponentsSpec extends BaseSpec with TestConstants {
-
   val h1English: String = "Heading level 1 text"
```

Comment:
I don't think the Welsh variables, such as h1Welsh, are used anymore. If not please remove.

Reply:
removed

Reply:
removed